arangosh/Shell/V8ClientConnection.cpp

```diff
@@ -1893,10 +1879,14 @@ v8::Local<v8::Value> parseReplyBodyToV8(fu::Response const& response,
 v8::Local<v8::Value> translateResultBodyToV8(fu::Response const& response,
                                              v8::Isolate* isolate) {
   auto responseBody = response.payload();
-
-  if ((response.contentEncoding() == fuerte::ContentEncoding::Identity) ||
-      response.isContentTypeHtml() ||
-      response.isContentTypeText()) {
+  if ((responseBody.size() > 0) &&
```

Comment:
This change means that for any response body of length 0, we will create a Buffer as the result and no empty string. Is that intentional?

Reply:
yes, this was the original behaviour, tests asserted this.

Reply:
removed.